source/kernel/src/services/serial_mux.rs

```diff
@@ -40,6 +40,21 @@ pub enum WellKnown {
     PsuedoKeyboard = 2,
     /// A bidirectional for binary encoded tracing messages
     BinaryTracing = 3,
+
+    /// An interactive forth shell (1/4)
```

Comment:
nit, take it or leave it: it could be worth documenting the difference between `PseudoKeyboard` (just input, used for the graphical shell) and `ForthShell{N}` (which is bidirectional)?